### DISCUSSION TITLE
Custom path for controlnet models folder in A1111?

### DISCUSSION THREAD

**Discussion by suonnon** [Mikubill/sd-webui-controlnet#2648] [Q&A] [answered]
Hi mate,

Is there a way to set the custom path for the controlnet models folder? Because I want to use one folder for A1111, ComfoUI, InvokeAI

**Comment by huchenlei** [comment, marked as answer]
There are commandline args to help you do that. See https://github.com/Mikubill/sd-webui-controlnet#command-line-arguments

**Reply by suonnon** [reply]
Thank you so much!!!
